deployments/examples/ocis_traefik/README.md

```diff
@@ -0,0 +1,27 @@
+# Deployment scenario ocis with traefik
+
+## Setup on server
+
+*	Clone ocis repository
+
+	`git clone https://github.com/owncloud/ocis.git`
+
+*	Copy example folder to /opt
+
+	`cp deployment/examples/ocis_traefik /opt/`
+
+*	Overwrite OCIS_DOMAIN in .env with your.domain.com
+
+	`sed -i 's/ocis.domain.com/your.domain.com/g' /opt/ocis_traefik/.env`
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 1 space](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6324067)